### DISCUSSION TITLE
Adjusting base font size and having rest of fonts depend on that (rem)

### DISCUSSION THREAD

**Discussion by danielfdickinson** [adityatelange/hugo-PaperMod#530] [Ideas]
While looking at the more modern approach to #496 I realized that PaperMod uses a lot of hard-coded font-sizes, defined in pixels (px). I'd like to be able to adjust the base font size and have all other fonts adjust relative to that. That's not difficult with modern CSS, but I am thinking that making font sizes relative to the base font (using rem) is a change that should be separate from the logic for determining the base font size based on viewport size that I also want to add.

My questions are whether there any objection to a PR that makes all font sizes relative to the base font size (and makes the base font size a var in ``templates-vars.css``, and does this make sense to others?

Also, to keep things looking good it would be needed to relativize other things (like gaps and margins). I'm wondering how that ought to be organized into commits, for ease of review and so on.

**Comment by pixelfabs** [comment]
> My questions are whether there any objection to a PR that makes all font sizes relative to the base font size (and makes the base font size a var in `templates-vars.css`, and does this make sense to others?

Excellent idea. A base font-size is definitely needed, since there's a lot of difference in font-size between font-families. 16px is not 16px in every font-family.

**Comment by danielfdickinson** [comment]
I've create a draft PR at #531 . Please take a look.
